server/routes/templates.ts

```diff
@@ -326,10 +326,25 @@ async function applicationBranchHandler(request: express.Request, response: expr
 			savedValue = undefined;
 		}
 		question["value"] = savedValue ? savedValue.value : "";
+
+		if (questionBranch.text) {
+			let textContent: string = questionBranch.text.filter(text => text.for === question.name).map(text => {
+				return `<${text.type}>${sanitize(text.content)}</${text.type}>`;
```

Comment:
Would a library be overkill here? Since we're escaping *all* HTML input instead of transforming or allowing only certain HTML tags, wouldn't replacing the `<`, `>`, and `&` characters be sufficient?